packages/web-components/src/components/va-process-list/va-process-list-item.scss

```diff
@@ -9,6 +9,7 @@
 
 .usa-process-list__heading-eyebrow {
   color: var(--uswds-system-color-gray-cool-70);
```

Comment:
Can this variable be a `vads-` variable? I think technically we should probably be using our own mapped tokens.